packages/cli-kit/src/private/node/ui/components/FullScreen.tsx

```diff
@@ -7,26 +7,32 @@ import React, {useEffect, useState} from 'react'
  * - You want to respond to the resize event of the terminal. Whenever the user resizes their terminal window the output's height and width will be recalculated and re-rendered properly.
  */
 const FullScreen: React.FC = ({children}): JSX.Element => {
+  const {stdout} = useStdout()
+
+  if (!stdout) {
+    throw new Error('Could not find stdout')
+  }
```

Comment:
When does `!stdout` happen? If we don't know when it happens, what do you think about assuming `stdout` has a value?

Reply:
Yeah I don't foresee any cases in which it doesn't exist. This `if` is just there so TS understand that all calls after this are done to something that is not `undefined`. The alternative would be to do `stdout!.yyy` all the time. Would you prefer another way to accomplish the same thing? Maybe reassigning?

Reply:
I'd go the reassigning path to eliminate some verbosity.